vello/src/scene.rs

```diff
@@ -469,3 +618,30 @@ impl<'a> PathBuilder<'a> {
         }
     }
 }
+
+fn affine_to_f32(affine: &Affine) -> [f32; 6] {
+    let c = affine.as_coeffs();
+    [
+        c[0] as f32,
```

Comment:
This code can use [`[T;N]::map()`](https://doc.rust-lang.org/std/primitive.array.html#method.map). That's not important for this PR

Reply:
oh, const generics ftw!